### DISCUSSION TITLE
What is -K doing?

### DISCUSSION THREAD

**Discussion by AjazElahi** [madMAx43v3r/chia-plotter#833] [Q&A]
I've seen in the update we now have: 
 -K, --rmulti2 arg    Thread multiplier for P2 (default = 1)
 
 What does this do exactly? I'm wondering whether it would benefit me to use it but I'm not sure how. I have 2x E5-2640v3 processors giving total 16C/32T and I'm running -r 32 without trouble, making plots in 45min

**Comment by Nenco79** [comment]
You should try -r16 -K 2 and report back

**Comment by gryan315** [comment]
-K multiplies the -r value for phase 2. Phase 2 seems to have the fastest time when -r = logical cores, but phase 3 (dominant for time) is fastest with -r = physical cores. -K 2 allows both phase 2 and 3 to operate at their fastest -r settings simultaneously. It's not a huge gain, but it shaved an extra minute off of my previously fastest settings.
